src/actions/browser-actions/launch-browser/launch-browser.ts

```diff
@@ -1,7 +1,7 @@
-import { getChromePath, getChromeCanaryPath } from '../../../utils';
-import { chromium, Browser, firefox, webkit } from 'playwright';
+import {getChromePath, getChromeCanaryPath, getEdgePath} from '../../../utils';
+import {chromium, Browser, firefox, webkit} from 'playwright';

```

Comment:
Hi @osolomin90,

You have introduced a subtle change. 

Unless I am wrong, it seems to me that your IDE does not comply with the prettier/eslint rules of the repo.

The `import {getChromePath, getChromeCanaryPath, getEdgePath}` should be `import { getChromePath, getChromeCanaryPath, getEdgePath }`